src/instructlab/configuration.py

```diff
@@ -171,8 +173,9 @@ class _serve(BaseModel):
     model_path: StrictStr
 
     # additional fields with defaults
-    host_port: StrictStr = "127.0.0.1:8000"
-    backend: str = ""  # we don't set a default value here since it's auto-detected
+    host_port: StrictStr
+
+    backend: str
```

Comment:
Above comment also needs to be changed
```suggestion
    host_port: StrictStr
    backend: str
```